pkg/controller/controller_clusterserviceclass.go

```diff
@@ -55,7 +55,7 @@ func (c *controller) clusterServiceClassDelete(obj interface{}) {
 // reconciliation loop for ClusterServiceClass. ClusterServiceClasses are primarily
 // reconciled in a separate flow when a ClusterServiceBroker is reconciled.
 func (c *controller) reconcileClusterServiceClassKey(key string) error {
-	plan, err := c.clusterServiceClassLister.Get(key)
+	clusterserviceclass, err := c.clusterServiceClassLister.Get(key)
```

Comment:
Can you change the name to `class` or at least `clusterServiceClass`?

Reply:
ok, thanks for review.